graylog2-server/src/main/java/org/graylog2/alerts/types/FieldContentValueAlertCondition.java

```diff
@@ -88,10 +88,9 @@ public ConfigurationRequest getRequestedConfiguration() {
     public static class Descriptor extends AlertCondition.Descriptor {
         public Descriptor() {
             super(
-                "Field Content Value Alert Condition",
+                "Field Content Alert Condition",
                 "https://www.graylog.org/",
-                "This condition is triggered when the aggregated value of a field is higher/lower than a defined "
-                + "threshold for a given time range."
+                "This condition is triggered when the content of messages is equal to a defined value."
```

Comment:
Yes, you are right, conditions were not pluggable before 2.2, and I also don't think we have any condition plugins around to test.

The `human_name` is only used in here as far as I know: https://github.com/Graylog2/graylog2-server/blob/master/graylog2-web-interface/src/components/alertconditions/AlertConditionForm.jsx#L77-L77
Condition plugins may provide some custom UI for the create/edit form, but if they don't we use `human_name` to give some context inside the generic configuration form. I don't really like how the forms look at the moment, so we can probably display the `human_name` somewhere else too.